validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -284,63 +286,71 @@ def validate_on_chain_rules(self, blkw, prev_state_root):
         return True
 
     def validate_block(self, blkw, consensus, chain_head=None, chain=None):
+        if blkw.status == BlockStatus.Valid:
```

Comment:
The changes in this commit do not reflect the simplicity of the commit message.  It is thus difficult to determine the intent in some of these changes.

Reply:
I split the commit up into two and added more information to this part of the change.